QRCoder/Builders/Renderers/IStreamRenderer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.IO;
+
+namespace QRCoder.Builders.Renderers
+{
+    public interface IStreamRenderer
+    {
+        MemoryStream ToStream();
+    }
```

Comment:
Concept is that all byte-array-type renderers (or ones that can serialize to a byte array) implement this interface.  If they use a `MemoryStream` under the hood, it's a direct pass-through.  If they generate a byte array directly, then wrapping it in a `MemoryStream` is quite inexpensive, as it does not duplicate the entire byte array.